patterns/1-initial/outsourced-development-ecosystem.md

```diff
@@ -0,0 +1,80 @@
+## Title
+
+Outsourced Development Ecosystem
+
+## Patlet
+
+The burdens of existing contracts in an outsourced environment constraints the success of the
+InnerSource initiative. Outsourced companies may see this as a risky situation as they have to
+way to share knowledge, resources, and expertise with competitors within the mother company.
+The legal contracts typically forces to develop software in a very specific and silo-ed way.
+Redefining these rules and allowing outsourced companies would bring a more InnerSource-friendly
+way of working.
+
+## Problem
+
+Current outsourced environments in large corporations prevent those suppliers to work in an
+InnerSource way. This includes a transparent, collaborative, and community-oriented way.
+Beyond the implications of the cultural change, and process focus, most of the limitations are
+given by the existing legal framework that forces the outsourced development to charge a fee
+per hour and within a very specific project. There are not specific agreements to allow
+a collaborative way of working across the organization and across outsourced companies or contractors.
+
+## Context
+
+Large corporation with a big variety of outsourced companies and contractors that produce part
+of the code base of this company. Each company or contractor has its own legal contract in place
+that states the goals, the pricing schema, and the expected outcome and output for the organization.
+
+No collaboration happens at the outsourced company or contractor level and each of them are working
+in a silo-based mode with no interactions with others, but the mother company.
+
+
+## Forces
+
+* Legal teams and outsourced companies trying to avoid risk exposure and responsibilities.
+* Legal implications and responsibilities when something goes wrong.
+* SLAs in place forcing behaviors.
+* Managers losing control of the situation by having more collaboration across companies borders
+
+## Solutions
+
+* Governance model:
+  * Better definition of who owns the code and the data produced and if possible owned by the
+    mother company.
+  * Definition of checks and balances of who is responsible for what action (RACI) and SLAs against PRs.
+  * Definition of tools and processes so suppliers do not bring extra complexity to other suppliers.
+* Flexibility in the legal contracts to allow them to contribute with other companies,
+  even when they are competitors.
```

Comment:
Is there, in this solution space, an aspect of a business case that might help drive the adoption of this approach? To fight against the Manager fear of loss of control and the thought that each silo might worry about working with their (internal) competitors?